doc/classes/Geometry3D.xml

```diff
@@ -73,6 +73,17 @@
 				Given the two 3D segments ([param p1], [param p2]) and ([param q1], [param q2]), finds those two points on the two segments that are closest to each other. Returns a [PackedVector3Array] that contains this point on ([param p1], [param p2]) as well the accompanying point on ([param q1], [param q2]).
 			</description>
 		</method>
+		<method name="get_triangle_barycentric_coords">
+			<return type="Vector3" />
+			<param index="0" name="point" type="Vector3" />
+			<param index="1" name="v0" type="Vector3" />
+			<param index="2" name="v1" type="Vector3" />
+			<param index="3" name="v2" type="Vector3" />
+			<description>
+				Returns a [Vector3] containting weights based on how close a 3D position ([param point]) is to a triangle's different vertices ([param v0], [param v1] and [param v2]). This is useful for interpolating between the data of different vertices in a triangle. One example use case is using this to smoothly rotate over a mesh instead of relying solely on face normals.
```

Comment:
```suggestion
				Returns a [Vector3] containing weights based on how close a 3D position ([param point]) is to a triangle's different vertices ([param v0], [param v1] and [param v2]). This is useful for interpolating between the data of different vertices in a triangle. One example use case is using this to smoothly rotate over a mesh instead of relying solely on face normals.
```

Reply:
Fixed, thanks.